upstream-community-operators/postgresql/postgresql.package.yaml

```diff
@@ -3,3 +3,7 @@ defaultChannel: stable
 channels:
   - name: stable
     currentCSV: 'postgresoperator.v4.4.0'
+  - name: original_40
+    currentCSV: 'postgresoperator.v4.0.1'
+  - name: original_41
+    currentCSV: 'postgresoperator.v4.1.0'
```

Comment:
```suggestion
  - name: original_40
    currentCSV: 'postgresoperator.v4.0.1'
  - name: original_41
    currentCSV: 'postgresoperator.v4.1.0'
  - name: original_42
    currentCSV: 'postgresoperator.v4.2.2'
  - name: original_43
    currentCSV: 'postgresoperator.v4.3.2'
```

Reply:
Hi @jkatz. We are in the replaces mode. You can set `replaces` in specific CSV and point to previous version, so update is possible. If you don't specify replaces, it means version X has big changes and does not allow replacing version Y. So you can start new channel.
Maybe you just forgot to put replaces to versions `4.0.1` and `4.1.0`. Just put it there and all versions stay in stable channel. So we don't need to create new channels.
Our PR is just technically correct workaround, but only you know what if everything can be replaced by higher version or not. If replaces is missing somewhere, we have 2 options. Create new channel or add replaces where it is missing. The last option is better, but only possible from someone who knows operator better than me:)

Reply:
@J0zi - I believe technically they can all be replaced, but the upgrade process has simplified from 4.2 onward.

That said, I'm OK for the other option, adding replaces to the older ones so we don't need a separate channel. Would you like me to submit a PR on that?